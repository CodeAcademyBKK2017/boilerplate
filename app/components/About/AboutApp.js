import React from 'react';
import {
  Text
} from 'react-native';
  
export default class MyNotificationsScreen extends React.Component {
    static navigationOptions = {
      tabBarLabel: 'APP'
    };
    render () {
      return (
        <Text>APP</Text>
        // <Button
        //   onPress={() => this.props.navigation.goBack()}
        //   title='Go back home'
        // />
      );
    }
}
