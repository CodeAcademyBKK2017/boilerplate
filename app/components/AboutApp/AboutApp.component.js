import Icon from 'react-native-vector-icons/FontAwesome';
import React, {Component} from 'react';
import styles from './AboutApp.style';
import Touchable from 'react-native-platform-touchable';

import {Text, View} from 'react-native';

export default class AboutAppScreen extends Component {

  static navigationOptions = ({navigation}) => {
    const toggleDrawer = (navigation) => () => navigation.navigate('DrawerToggle');
    return {
      title: 'About App',
      headerLeft: <Touchable onPress={toggleDrawer(navigation)}><Icon name='bars' size={24} /></Touchable>
    };
  }
  
  render () {
    return (
      <View style={styles.container}>
        <Text>About App</Text>
      </View>
        
    );
  }
  
}