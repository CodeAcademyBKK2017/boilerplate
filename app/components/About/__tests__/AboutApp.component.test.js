import 'react-native';
import AboutApp from '../AboutApp.component';
import React from 'react';

// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer';

describe('AboutApp', () => {
  it('renders correctly', () => {
    const snapshot = renderer.create(<AboutApp/>).toJSON();
    expect(snapshot).toMatchSnapshot();
  });
});
