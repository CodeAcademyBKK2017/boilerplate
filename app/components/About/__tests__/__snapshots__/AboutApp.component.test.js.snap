// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AboutApp renders correctly 1`] = `
<View>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
  >
    About App
  </Text>
</View>
`;
